sphinx/tutorial/rose/furthertopics/rose-arch.rst

```diff
@@ -139,12 +139,12 @@ You have now created a suite that defines three tasks:
 
 ``make_files``
    Sets up the files and ``ARCHIVING/`` directory for ``archive_files_rsync/``
-   and ``archive_files_scp/`` to "archive", move, date to.
+   and ``archive_files_scp/`` to "archive", move, data to.
 ``archive_files_rsync``
-   "Archives", rsync's files to the ``ARCHIVING/`` folder in the
+   "Archives" ``rsync``'s files to the ``ARCHIVING/`` folder in the
    ``$ROSE_DATA/`` directory.
 ``archive_files_scp``
-   "Archives", scp's the renamed files and moves them to the ``ARCHIVING/``
+   "Archives" ``scp``'s the renamed files and moves them to the ``ARCHIVING/``
```

Comment:
'the renamed' still renders this sentence nonsensical so with please quickly amend (put in brackets or remove 'the' etc).